### ISSUE TITLE
Can't use https://textfsm.nornir.tech to import textfsm templates

### ISSUE THREAD

**Issue by hanunes** [tbotnz/netpalm#187]
I used to be able to import textfsm templates from https://textfsm.nornir.tech. Now with the update of the website, this is no longer possible, or is it?

If it is possible please let us know how.

Thanks in advance.

**Comment by tbotnz** [comment]
Hi there,

Apologies i got tired of paying for the storage for all of the templates on that site. It might be possible using netpalm admin (though ive forgotten if it supports upload with tfsm).

If you really really need it let me know and we can try and work something out

Thanks
Tony

**Comment by hanunes** [comment]
Hi,

I've managed to work something out on my end. So basically I have a backend now which interfaces with netpalm to be able to easily add/edit/remove templates. Does all the base64 encoding/decoding automatically, so in one call on my APP it does all I needed to do previously by hand.

On another note, I've stopped using textfsm completely, ttptemplates are much more powerfull!

Thanks for taking the time though.
